king_phisher/client/dialogs/campaign_selection.py

```diff
@@ -154,7 +192,11 @@ def load_campaigns(self):
 				(hlfg_color if is_expired else fg_color),
 				(html.escape(campaign.description, quote=True) if campaign.description else None)
 			))
-		self.gobjects['label_campaign_info'].set_text("Showing {0:,} Campaign{1}".format(len(store), ('' if len(store) == 1 else 's')))
+		self.gobjects['label_campaign_info'].set_text("Showing {0} of {1:,} campaign{2} found in the database.".format(
+																													len(self._tv_model_filter),
```

Comment:
To be consistent with the other indentations of this style within the project, this line should be indented exactly one more than the line before it. The closing parenthesis should be aligned with the start of the line before this one.